src/status_im/contexts/wallet/create_account/edit_derivation_path/view.cljs

```diff
@@ -1,16 +1,51 @@
 (ns status-im.contexts.wallet.create-account.edit-derivation-path.view
   (:require
     [quo.core :as quo]
+    [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [react-native.safe-area :as safe-area]
     [reagent.core :as reagent]
+    [status-im.constants :as constants]
     [status-im.contexts.wallet.common.temp :as temp]
     [status-im.contexts.wallet.common.utils :as utils]
     [status-im.contexts.wallet.create-account.edit-derivation-path.style :as style]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn path-format-sheet
```

Comment:
looks like this page has many bottom sheets. Perhaps we can keep this file quite trim as there will be quite a lot happening in this page.
perhaps
`status-im.contexts.wallet.create-account.edit-derivation-path.view`
`status-im.contexts.wallet.create-account.edit-derivation-path.path-format` (maybe we can export from there as `sheet` so the usage is `path-format/sheet`

as there will likely also be:
`status-im.contexts.wallet.create-account.edit-derivation-path.select-address` (select address bottom sheet)

Reply:
at the very least we should mark this private anyway 👍